bin/gdb_wrapper

```diff
@@ -78,51 +79,64 @@ require 'ruby-debug-ide/attach/util'
 require 'ruby-debug-ide/attach/native_debugger'
 require 'ruby-debug-ide/attach/process_thread'
 
-debugger = choose_debugger(options.ruby_path, options.pid, options.gems_to_include, debugger_loader_path, argv)
+pids = get_child_pids(options.pid.to_s)
+attach_threads = Array.new
 
-trap('INT') do
-  unless debugger.exited?
-    $stderr.puts "backtraces for threads:\n\n"
-    process_threads = debugger.process_threads
-    if process_threads
-      process_threads.each do |thread|
-        $stderr.puts "#{thread.thread_info}\n#{thread.last_bt}\n\n"
-      end
-    end
-    debugger.exit
+pids.each_with_index do |pid, i|
+
+  if(i == 1)
```

Comment:
After the extraction of the logic below into methods this `each_with_index` and `if i == ...`, which are not beautiful, could be transformed to normal logic like "do something with the parent process and do something else with all its children" without any additional overhead.